spec/models/stripe_wrapper_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe StripeWrapper do
+  describe StripeWrapper::Charge do
+    describe ".create" do
+      it "makes a successful charge", :vcr do
+        token = Stripe::Token.create(
+          :card => {
+            :number => "4242424242424242",
+            :exp_month => 6,
+            :exp_year => 2018,
+            :cvc => "314"
+          },
+        ).id
+
+        response = StripeWrapper::Charge.create(
+          amount: 999,
+          card: token,
+          description: 'a valid charge' 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.